src/Locations/Locations.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+
```

Comment:
Might seem like a minor note, but real-world engineers don't want to have to clean up after juniors: watch for extra whitespace left in. You have a few functions with extra blank lines left around.

(I used to leave those in there ALL THE TIME.)